src/data/queryParams.ts

```diff
@@ -58,9 +63,13 @@ export const useQueryDataIfPresent = (
   calcData: CalculatorData,
 ): CalculatorData => {
   const queryDataFiltered = pickBy(queryData, (v) => v !== undefined)
+
   if (isEmpty(queryDataFiltered)) {
     return calcData
   } else {
-    return { ...defaultValues, ...queryDataFiltered }
+    return sanitizeData(
+      queryDataFiltered,
+      true /* fillCustomIfScenarioMissing */,
```

Comment:
Is there a case where this parameter if `false`? If not, remove parameter?

Reply:
yes.

0. check out code from before the saved scenario UI change
1. run the calculator until you see a points total and something has been saved to local storage
2. hit the reset button to rewrite the thing in local storage with default data
3. note that there is no scenarioName in local storage, so we have the option of filling it with "custom" (allows old saved scenarios and old links to show up properly) or "" (allows reset form to show the typeahead to pick a scenario instead of directly launching you into a custom scenario). I've got it to pick "custom" if the local storage interaction type is nonempty (suggesting that the local storage contains a genuine scenario) or if the data is coming from query params. "" if local storage interaction type is empty.

Reply:
Ah, I see, missed the other call.